### ISSUE TITLE
Support for bagging_freq parameter

### ISSUE THREAD

**Issue by chaaland** [dmlc/xgboost#5757]
LightGBM supports a [bagging_freq parameter](https://lightgbm.readthedocs.io/en/latest/Parameters.html#bagging_freq). From the docs I do not see an equivalent ability in XGBoost.

**Comment by hcho3** [comment]
Right now, if you set `subsample` to be less than 1.0, the data gets sampled every iteration.

**Comment by chaaland** [comment]
Right. The request would be to make the number of iterations before resampling a user adjustable parameter. As in the LightGBM parameter I linked
